src/status_im2/contexts/shell/bottom_tabs.cljs

```diff
@@ -17,35 +18,44 @@
    :overlay-color :transparent})
 
 (defn bottom-tab
-  [icon stack-id shared-values notifications-data]
-  [bottom-nav-tab/bottom-nav-tab
-   (assoc (get notifications-data stack-id)
-          :test-ID             stack-id
-          :icon                icon
-          :icon-color-anim     (get
-                                shared-values
-                                (get shell.constants/tabs-icon-color-keywords stack-id))
-          :on-press            #(animation/bottom-tab-on-press stack-id)
-          :accessibility-label (str (name stack-id) "-tab"))])
+  ([icon stack-id shared-values notifications-data]
+   [bottom-tab icon stack-id shared-values notifications-data nil])
+  ([icon stack-id shared-values notifications-data double-tap-gesture]
+   [bottom-nav-tab/bottom-nav-tab
+    (cond-> (assoc (get notifications-data stack-id)
+                   :test-ID             stack-id
+                   :icon                icon
+                   :icon-color-anim     (get
+                                         shared-values
+                                         (get shell.constants/tabs-icon-color-keywords stack-id))
+                   :on-press            #(animation/bottom-tab-on-press stack-id)
+                   :accessibility-label (str (name stack-id) "-tab"))
+      double-tap-gesture (assoc :double-tap-gesture double-tap-gesture))]))
 
 (defn bottom-tabs
   []
   [:f>
    (fn []
-     (let [notifications-data (rf/sub [:shell/bottom-tabs-notifications-data])
-           pass-through?      (rf/sub [:shell/shell-pass-through?])
-           shared-values      @animation/shared-values-atom
-           original-style     (style/bottom-tabs-container pass-through?)
-           animated-style     (reanimated/apply-animations-to-style
-                               {:height (:bottom-tabs-height shared-values)}
-                               original-style)]
+     (let [notifications-data          (rf/sub [:shell/bottom-tabs-notifications-data])
+           pass-through?               (rf/sub [:shell/shell-pass-through?])
+           shared-values               @animation/shared-values-atom
+           original-style              (style/bottom-tabs-container pass-through?)
+           animated-style              (reanimated/apply-animations-to-style
+                                        {:height (:bottom-tabs-height shared-values)}
+                                        original-style)
+           messages-double-tap-gesture (-> (gesture/gesture-tap)
+                                           (gesture/number-of-taps 2)
+                                           (gesture/on-start
+                                            (fn [_event]
+                                              (rf/dispatch [:messages-home-view/select-tab :recent]))))]
```

Comment:
Slightly better IMO would be to use a keyword with qualification, like `:tab/recent`. It's more "searchable" since it's not a common word that can appear in the middle of other Clojure vars or comments. But I understand the current keyword was already here before your PR, so no worries.

Reply:
Why not refactor when you can :)
Done, Thanks Icaro! :)